compass/landice/__init__.py

```diff
@@ -1,4 +1,5 @@
 from compass.mpas_core import MpasCore
+from compass.landice.tests.antarctic import Antarctic
```

Comment:
I'd prefer we call this new test group `Antarctica` instead of `Antarctic`.  I realize that's kind of an annoying, picky comment because it requires touching a lot of different files, but this could end up being a large and often used test group, so I think it's worth requesting.